rasa/nlu/selectors/response_selector.py

```diff
@@ -244,30 +256,20 @@ def label_key(self) -> Text:
         return LABEL_IDS
 
     @staticmethod
-    def model_class() -> Type[RasaModel]:
-        return DIET2DIET
+    def model_class(use_text_as_label: bool) -> Type[RasaModel]:
+        if use_text_as_label:
+            return DIET2DIET
+        else:
+            return DIET2BOW
```

Comment:
This and the almost the whole of `DIET2BOW` class was needed because metrics are tightly coupled with the model architecture right now. See https://github.com/RasaHQ/research/issues/112